src/components/navBar/navBar.module.scss

```diff
@@ -1,35 +1,137 @@
-$offset: 10px;
-$thickness: 3px;
+$font-stack: "Roboto", sans-serif;
+$color-white: #ffffff;
+$color-primary: #1d1283;
+$color-highlight: #49a82e;
+$color-highlight2: #e30464;
+$color-boxshadow: rgba(0, 0, 0, 0.5);
 
 .navBar {
+  font: 700 16px $font-stack;
+  margin-bottom: 15px;
+
+  .navBarMenu {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    align-items: center;
+    background-color: $color-primary;
+
+    .navBarLoginLi {
+      margin-left: auto;
+    }
+
+    .navBarLogoLi {
+      padding: 12px 20px;
+    }
+  }
+}
+
+.homeTab,
+.loginBtn,
+.hamburger {
+  display: none;
+}
+
+.btnLogin {
   display: flex;
-  width: 100%;
-  background-color: #1d1283;
   align-items: center;
-  overflow: hidden;
-  flex-wrap: wrap;
+  margin-right: 23px;
+  cursor: pointer;
+  border-radius: 6px;
+  padding: 5px;
+  background-color: $color-primary;
+  color: $color-white;
+  border: 2px solid $color-white;
 
-  a {
-    display: block;
-    color: #ffffff;
-    font-weight: bolder;
-    text-align: center;
-    padding: 35px 2*($offset+$thickness)-1;
-    text-decoration: none;
+  .btnLoginText {
+    margin-right: 4px;
+  }
+}
+
+.userGreet {
+  display: flex;
+  align-items: center;
+  margin-right: 23px;
+  cursor: pointer;
+  color: $color-white;
+  padding: 0;
+  text-decoration: none;
+  background: transparent;
+  border: none;
+  font-size: 1rem;
+  font-weight: 700;
+
+  .userProfilePic {
+    border-radius: 50%;
+  }
+
+  .userGreetMsg {
+    margin-right: 10px;
+  }
+}
+
+.navBarLink {
+  color: $color-white;
+  margin: 0px 26px;
+  text-decoration: none;
+}
+
+.activeTab {
```

Comment:
If this is the active link then the name should reflect LINK, not TAB. As well as, use SASS nesting feature to handle active class

Eg:

```
.nav{
    a{
        &.active{
            color: red;
        }
    }
}
```

Reply:
Done